ansible/roles/tldraw-client-core/templates/tldraw-client-deployment.yml.j2

```
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tldraw-client-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: tldraw-client
+spec:
+  replicas: {{ TLDRAW_CLIENT_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: tldraw-client
```

We add no more labels after hpi-schul-cloud/schulcloud-server#4517 was merged

```suggestion
  labels:
    app: tldraw-client
    app.kubernetes.io/part-of: schulcloud-verbund
    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
    app.kubernetes.io/name: tldraw-client
    app.kubernetes.io/component: tldraw
    app.kubernetes.io/managed-by: ansible
    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
```